<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,10,1,1,0">
      <item row="7" column="0">
       <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,0,0,0,0,0" columnstretch="2,0,0,0,0">
        <item row="3" column="3">
         <layout class="QHBoxLayout" name="random_reward_percent_horizontalLayout" stretch="1,1">
          <item>
           <widget class="QLabel" name="random_reward_percent_label">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
            </property>
            <property name="text">
             <string>Random reward percent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="random_reward_percent_spinBox"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1" colspan="3">
         <layout class="QGridLayout" name="navigation_gridLayout_3" columnstretch="1,1">
          <item row="0" column="1">
           <widget class="QPushButton" name="next_pushBtn_3">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="back_pushBtn_3">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1" rowspan="2">
         <layout class="QGridLayout" name="iti_min_max_values_gridLayout_3">
          <item row="2" column="0">
           <widget class="QLabel" name="min_iti_label_3">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
            </property>
            <property name="text">
             <string>min: </string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="max_iti_spinBox"/>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="min_iti_spinBox"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="max_iti_label_3">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
            </property>
            <property name="text">
             <string>max: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QComboBox" name="behaviors_comboBox_3"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="iti_label_3">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
          </property>
          <property name="text">
           <string>ITI:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="end_def_comboBox_3"/>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="end_def_label_3">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
          </property>
          <property name="text">
           <string>End definition:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <layout class="QHBoxLayout" name="end_def_value_horizontalLayout_3">
          <item>
           <widget class="QLabel" name="end_by_label_3">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
            </property>
            <property name="text">
             <string>time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="time_spinBox_3"/>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="iti_def_horizontalLayout_3" stretch="0,0">
          <item>
           <widget class="QRadioButton" name="behavior_iti_radioBtn_3">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
            </property>
            <property name="text">
             <string>behavior</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="random_iti_radioBtn_3">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
            </property>
            <property name="text">
             <string>random</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="def_trial_horizontalLayout" stretch="1,0,1">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="order_horizontalLayout" stretch="1,1">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="trials_order_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
            </property>
            <property name="text">
             <string>Order:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="trials_order_comboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttons_to_set_trial_horizontalLayout_2" stretch="0,0,0">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="add_trial_pushButton">
            <property name="text">
             <string>Add</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="edit_trial_pushButton">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_trial_pushButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="session_name_and_date_horizontalLayout" stretch="1,0,1,1">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="session_name_label">
          <property name="font">
           <font>
            <family>Gabriola</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
          </property>
          <property name="text">
           <string>Session name:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="session_name">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="date_label">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
          </property>
          <property name="text">
           <string>Date:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="date_value_label">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
          </property>
          <property name="text">
           <string>13-April-2022</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="headline_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>134</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Gabriola</family>
          <pointsize>55</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 55pt &quot;Gabriola&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Create a session</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="choose_template__horizontalLayout" stretch="0,2,0">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="choose_template_pushButton_2">
          <property name="text">
           <string>Choose Template</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QGridLayout" name="trials_gridLayout_3" rowstretch="1,0" columnstretch="1">
        <item row="1" column="0">
         <widget class="QTableWidget" name="trials_tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>538</width>
            <height>130</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <column>
           <property name="text">
            <string>type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>parameters</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="trials_label">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
          </property>
          <property name="text">
           <string>Trials: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="subject_and_experimenter_name_horizontalLayout" stretch="1,0,1,0">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="subject_id_label">
          <property name="font">
           <font>
            <family>Gabriola</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
          </property>
          <property name="text">
           <string>Subject ID: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="sunject_name">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="experimenter_name_label">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Gabriola&quot;;</string>
          </property>
          <property name="text">
           <string>Experimentor name:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="exp_name">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
